commit 214859e3567ea9def85305e4f021a5d407e1ccfe
Author: Jose Dapena Paz <jdapena@igalia.com>
Date:   Tue Jan 23 10:56:36 2024 +0000

    IWYU: missing include for usage of std::bitset in resolution_monitor.cc
    
    Bug: 957519
    Change-Id: Ia538bbca63105397963632d2a145886e256efeb6
    Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/5190545
    Commit-Queue: José Dapena Paz <jdapena@igalia.com>
    Reviewed-by: Henrik Boström <hbos@chromium.org>
    Cr-Commit-Position: refs/heads/main@{#1250732}

diff --git a/third_party/blink/renderer/platform/peerconnection/resolution_monitor.cc b/third_party/blink/renderer/platform/peerconnection/resolution_monitor.cc
index 5d6a6495045c4..33135a1b22473 100644
--- a/third_party/blink/renderer/platform/peerconnection/resolution_monitor.cc
+++ b/third_party/blink/renderer/platform/peerconnection/resolution_monitor.cc
@@ -4,6 +4,8 @@
 
 #include "third_party/blink/renderer/platform/peerconnection/resolution_monitor.h"
 
+#include <bitset>
+
 #include "base/containers/span.h"
 #include "base/logging.h"
 #include "base/memory/ptr_util.h"
